### ISSUE TITLE
Mediapart login check fails with Wallabag 2.2.3

### ISSUE THREAD

**Issue by ghostofkendo** [wallabag/wallabag#3424]
### Issue details

I put my Mediapart credentials in vendor/j0k3r/graby-site-config/mediapart.fr.txt but fetching an article does not work: it creates an empty entry in wallabag.
According to [var/logs/prod.log](https://gist.github.com/ghostofkendo/da51ca7a1c1c24221cada04bc7f559a5), the POST request to https://www.mediapart.fr/login_check fails with a 400 Bad Request error.

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.2.3
* How did you install wallabag? Via the [Yunohost app](https://github.com/YunoHost-Apps/wallabag2_ynh)
* Last wallabag version that did not exhibit the issue (if applicable): n/a
* php version: 5.6.30-0+deb8u1
* OS: Debian 8 Jessie
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MariaDB

### Steps to reproduce/test case

1. [Enable debug level logging in wallabag](https://doc.wallabag.org/en/user/errors_during_fetching.html#enable-log-to-help-us-identify-the-problem)
2. Enter the credentials of an active Mediapart account in vendor/j0k3r/graby-site-config/mediapart.fr.txt
3. Try to add any article from Mediapart in wallabag (e.g. https://www.mediapart.fr/journal/france/191117/de-quoi-mediapart-est-il-le-nom)

**Comment by bdunogier** [comment]
The credentials must be added to your application's config, @ghostofkendo. In {{app/config/parameters.yml}}:

```yaml
parameters:
    sites_credentials:
        mediapart.fr:
            username: *****
            password: *****
```

**Comment by ghostofkendo** [comment]
I reverted the changes I had made in `vendor/j0k3r/graby-site-config/mediapart.fr.txt`, put my credentials in `app/config/parameters.yml` instead, and it works now.
Thanks @bdunogier 

Maybe this should be made explicit in the documentation on [restricted access articles](https://doc.wallabag.org/en/user/articles/restricted.html)?
This would avoid mistakes like mine.

**Comment by bdunogier** [comment]
No problem. In regards to the doc, it describes the "official" way (using
the UI), but not the steps you describe here. Am I correct ?

**Comment by nicosomb** [comment]
The official way is valid only for 2.3. Sorry, the doc is in the future (I know, we have to work on our project organization...)

**Comment by ghostofkendo** [comment]
Yes, the doc currently only describes the "official", improved way to configure credentials via the web UI, which will be available in the not-released-yet version 2.3.

I could propose changes to the doc, explaining how to configure it in wallabag 2.2, if you would like it.

**Comment by nicosomb** [comment]
Hello @ghostofkendo 

As the 2.3 is released, I close this issue. Feel free to comment if necessary.
